include/Vect.hpp

```diff
@@ -0,0 +1,326 @@
+#ifndef VECT_HPP_
```

Comment:
WHo THE FuCK WRoTE ThIS PIeCE oF SHiT? i SURe HOPe He LeARNs hE lEarns C++14! /s